src/coreComponents/mesh/generators/MeshGeneratorBase.hpp

```diff
@@ -41,7 +38,7 @@ class CellBlockManager;
  *  @brief The MeshGeneratorBase class provides an abstract base class implementation for different mesh types.
  *	   The MeshGeneratorBase is the Group specialization for different type of mesh handling.
  */
-class MeshGeneratorBase : public dataRepository::Group
+class MeshGeneratorBase : public MeshBase
```

Comment:
Doing so, you make the (yet to become) `generators` module depend on `mesh` while we have been willing to de the exact opposite: `generators` would become a "somehow" independent "source" of meshing informations.

Reply:
I can rename it _GeneratorBase_. But that’s not the path you are drawing. I would be happy to revamp my work in a sibling modèle of the generator one called « data repository »